.github/workflows/pull-request.yml

```diff
@@ -50,8 +92,11 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body: |
             ### Added or changed icons
+            ${{ steps.generate-1px-stroke-width.outputs.body }}
+            ${{ steps.generate-2px-stroke-width.outputs.body }}
+            ${{ steps.generate-3px-stroke-width.outputs.body }}
```

Comment:
Nice to wrap these in a expander as well
```suggestion
            <details>
            <summary>Icon stroke widths</summary>
            ${{ steps.generate-1px-stroke-width.outputs.body }}
            ${{ steps.generate-2px-stroke-width.outputs.body }}
            ${{ steps.generate-3px-stroke-width.outputs.body }}
            </details>
```

Reply:
Why, they don't use up a lot of space and are nice to look at?

Reply:
hmm, true. But I've some ideas for more, (like with inverted background)
So I thought maybe we can keep this bot commenter a bit organized.

Reply:
We could ony show the 2px variant and wrap the other ones with details.

Reply:
Maybe let's add a row of 2px previews in a single row and behind a details element all icons in 1, 2 and 3px widths.

![image](https://user-images.githubusercontent.com/17746067/235843752-8f63ff44-ae16-4e1a-b26f-ef25034ff76f.png)

=> 

![image](https://user-images.githubusercontent.com/17746067/235843734-98dd511a-9826-4c38-a6be-3565ab0d0b8d.png)

Reply:
```suggestion
            ${{ steps.generate-2px-stroke-width.outputs.body }}
            <details>
            <summary>Preview stroke widths</summary>
            ${{ steps.generate-1px-stroke-width.outputs.body }}
            ${{ steps.generate-2px-stroke-width.outputs.body }}
            ${{ steps.generate-3px-stroke-width.outputs.body }}
            </details>
```